Sources/Toolbox/Extensions/UIView+AutoLayout.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  File.swift
+//  
+//
+//  Created by Oliver Krakora on 06.11.20.
+//
+
+import UIKit
+
+extension UIView {
+    
+    /// Adds `self` to `view` as a subview and applies constraints to all edges
+    /// - Parameter view: The superview to which the view should be added as a subview
+    /// - Parameter resizeToFit: If true the superview will be resized to the minimum required size to fulfill all constraints
+    /// - Parameter offset: The offset that should be applied to all edges of the subview
+    func pin(to view: UIView, offset: UIEdgeInsets = .zero) {
+        self.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(self)
+        
+        NSLayoutConstraint.activate([
+            leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: offset.left),
+            topAnchor.constraint(equalTo: view.topAnchor, constant: offset.top),
+            view.trailingAnchor.constraint(equalTo: trailingAnchor, constant: offset.right),
+            bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: offset.bottom)
```

Comment:
This will cause a negative offset.
replace with:
`view.bottomAnchor.constraint(equalTo: bottomAnchor, constant: offset.bottom`